src/App.tsx

```diff
@@ -15,36 +16,113 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  ABC = 'Sort alphabetically',
+  LENGTH = 'Sort by length',
+  RESET = 'Reset',
+}
+
+type DataFromServer = string;
+
+const handleSorting = function (
```

Comment:
The function's name should start with a verb to clearly indicate what the function does. Consider renaming `handleSorting` to something more descriptive, like `sortData` or `getSortedData`.